mcaleb808/myPharmBackend
# Setup Code Climate to run locally

---

#### Description of changes made

> This PR was to set the code climate CLI tool and its config file to check for code quality on a local machine by simply running the local CLI tool.
#### Type of change

> **Please select the relevant options**


#### How Has This Been Tested?

> **Please describe the tests that you ran to verify your changes**


#### How should this be manually tested?

> **Provide the steps to manually test the functionalities implemented in this PR**

- After cloning the [repo](https://github.com/mcaleb808/myPharmBackend.git), and `cd into the project`
- Check out this branch by `git checkout setup-codeclimate-locally`
- Run `yarn | npm install` to install the project dependencies
- Make sure to have your `.env` file set up correctly
- Run `yarn dev | npm run dev` to start the local server.

#### Checklist:


#### GitHub issue your changes fix 

Resolves #11 

#### Screenshots
<img width="708" alt="Screen Shot 2019-11-21 at 12 32 17" src="https://user-images.githubusercontent.com/29999424/69330199-2b864b00-0c5b-11ea-8c21-d5eb9dcace75.png">

<img width="773" alt="Screen Shot 2019-11-21 at 12 33 03" src="https://user-images.githubusercontent.com/29999424/69330209-2e813b80-0c5b-11ea-974d-b7fb7ed7cc2d.png">
